--- mantisbt-1.2.5/config_defaults_inc.php~	2011-04-05 20:24:17.000000000 +0200
+++ mantisbt-1.2.5/config_defaults_inc.php	2011-04-19 12:48:34.379687628 +0200
@@ -49,7 +49,7 @@
 	 * The default user name is "root".
 	 * @global string $g_db_username
 	 */
-	$g_db_username			= 'root';
+	$g_db_username			= 'mysql';
 	/**
 	 * Password for the specified user name. The default password is empty.
 	 * @global string $g_db_password
@@ -60,7 +60,7 @@
 	  * The default database name is "bugtracker".
 	  * @global string $g_database_name
 	  */
-	$g_database_name		= 'bugtracker';
+	$g_database_name		= 'mantis';
 
 	/**
 	 * Database Schema Name - used in the case of db2.
@@ -149,7 +149,7 @@
 	 * absolute path to your installation.  Requires trailing / or \
 	 * @global string $g_absolute_path
 	 */
-	$g_absolute_path		= dirname( __FILE__ ) . DIRECTORY_SEPARATOR;
+	$g_absolute_path		= '/usr/share/mantis' . DIRECTORY_SEPARATOR;
 
 	/**
 	 * absolute patch to your core files. The default is usually OK,
--- mantisbt-1.2.5/config_inc.php.sample~	2011-04-05 20:24:17.000000000 +0200
+++ mantisbt-1.2.5/config_inc.php.sample	2011-04-19 12:49:30.938939711 +0200
@@ -35,9 +35,9 @@
 
 # --- Database Configuration ---
 $g_hostname      = 'localhost';
-$g_db_username   = 'mantisdbuser';
+$g_db_username   = 'mysql';
 $g_db_password   = '';
-$g_database_name = 'bugtracker';
+$g_database_name = 'mantis';
 $g_db_type       = 'mysql';
 
 # --- Anonymous Access / Signup ---
